packages/gatsby-source-shopify/__tests__/fixtures/shopify-nodes/ShopifyProduct.json

```diff
@@ -1 +1 @@
d":"gid://shopify/Product/6966721052878","media___NODE":[],"variants___NODE":["435874ba-a8b0-5b29-a96a-8691e694bf3a","a0191127-d288-510d-94e5-14f7a6404454","cba433a6-1513-576f-97c0-80b36fb8afa6"],"metafields___NODE":["e3bdda06-d619-5445-8b4a-d1155678928c"],"collections___NODE":["5e2d818c-9edd-5d9a-9b8b-a87b59518efc"],"id":"1386d0f3-354b-584c-a6bb-d11936c8f6fb","internal":{"type":"ShopifyProduct","contentDigest":"c768cf13de234c3723e300b640e15c46","owner":"gatsby-source-shopify","counter":29},"children":[],"parent":null},{"createdAt":"2022-02-15T22:22:42Z","description":"Test Description","descriptionHtml":"Test Description","featuredImage":{"altText":null,"height":1280,"originalSrc":"https://cdn.shopify.com/s/files/1/0506/3521/0958/products/placeholder_aa389e41-a6d1-453e-ba2a-dae56c3993eb.png?v=1644963764","src":"https://cdn.shopify.com/s/files/1/0506/3521/0958/products/placeholder_aa389e41-a6d1-453e-ba2a-dae56c3993eb.png?v=1644963764","transformedSrc":"https://cdn.shopify.com/s/files/1/0506/3521/0958/products/placeholder_aa389e41-a6d1-453e-ba2a-dae56c3993eb.png?v=1644963764","width":1920,"shopifyId":"gid://shopify/ProductImage/30403196551374"},"featuredMedia":{"shopifyId":"gid://shopify/MediaImage/22755647652046"},"feedback":{"details":[],"summary":"This product couldn’t be published to Google"},"giftCardTemplateSuffix":"","handle":"test-product","hasOnlyDefaultVariant":false,"hasOutOfStockVariants":false,"isGiftCard":false,"legacyResourceId":"6950279086286","mediaCount":1,"onlineStorePreviewUrl":"https://atomic-digital-studio-test.myshopify.com/products/test-product","onlineStoreUrl":null,"options":[{"name":"Size","position":1,"values":["Small","Medium","Large"],"shopifyId":"gid://shopify/ProductOption/8894327619790"}],"priceRange":{"maxVariantPrice":{"amount":"300.0","currencyCode":"CAD"},"minVariantPrice":{"amount":"100.0","currencyCode":"CAD"}},"priceRangeV2":{"maxVariantPrice":{"amount":"3.0","currencyCode":"CAD"},"minVariantPrice":{"amount":"1.0","currencyCode":"CAD"}},"productType":"Test Type","publishedAt":"2022-03-14T21:26:01Z","requiresSellingPlan":false,"sellingPlanGroupsCount":0,"seo":{"description":null,"title":null},"status":"ACTIVE","storefrontId":"Z2lkOi8vc2hvcGlmeS9Qcm9kdWN0LzY5NTAyNzkwODYyODY=","tags":["test-tag"],"templateSuffix":"","title":"Test Product","totalInventory":3,"totalVariants":3,"tracksInventory":true,"updatedAt":"2022-03-14T21:26:01Z","vendor":"Gatsby","shopifyId":"gid://shopify/Product/6950279086286","media___NODE":["8fc15b46-8400-5778-b85c-05db5c009326"],"variants___NODE":["232cdd32-2de5-5bdd-a327-08d539ecd51e","266c6a92-078a-519d-8c03-257b4ed15b84","480d2d09-c1b1-5f7c-94a2-635e73e50ef4"],"metafields___NODE":["0f64cc58-bb8b-5173-851a-34fd9a5137a3"],"collections___NODE":["5e2d818c-9edd-5d9a-9b8b-a87b59518efc"],"id":"87b524a6-a707-5eb9-b193-059b3fcf96e5","internal":{"type":"ShopifyProduct","contentDigest":"469984af6fa3e64850c1cccc32d072b7","owner":"gatsby-source-shopify","counter":32},"children":[],"parent":null}]
```

Comment:
Same not about prettifying

Reply:
Addressed in https://github.com/gatsbyjs/gatsby/pull/39082/commits/2feb7371e217fcb915581ebf528b26b7ac43e12d